framework/logging.cpp

```diff
@@ -539,7 +539,7 @@ void logging_init_global_child()
     assert(sApp->current_fork_mode() == SandstoneApplication::child_exec_each_test);
 
     file_log_fd = real_stdout_fd = STDOUT_FILENO;
-#ifndef NDEBUG
+#ifdef DEBUG
```

Comment:
`#ifndef NDEBUG` was correct. `NDEBUG` is the C standard way of saying "no debugging" and is used by  `<assert.h>`. The double negative must stay.

See https://cigix.me/c17#7.2

Reply:
Gotcha. Oof.